GSA/digitalgov.gov
# Bug: Fixing 'News' display issue on author pages

## Summary

Creating a new partial for the 'See all News' button to address consistency both visually and in content.   



### Solution

Created a new partial file and then added it to all files that used the hard-coded format and created a separate style sheet.

Before

<img width="1158" alt="Screenshot 2024-04-03 at 4 20 17 PM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/6deae6ef-6f19-4c02-b4ea-c8a43b1c08aa"><br><br><br>
After
<img width="1063" alt="Screenshot 2024-04-03 at 4 20 06 PM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/d6a64e6e-3f79-4d95-994c-bbb81e2acbe8">
 

### How To Test

1. Vist preview link and check that the new partial is now styled as expected and homepage section does not have any regressions 

| Preview   | Live Site |
| -------- | ------- |
| [Homepage](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-footer-bug-update/) |  [Homepage](https://digital.gov/)  |
| [Authors page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-footer-bug-update//authors/dan-williams/) | [Authors page](https://digital.gov/authors/dan-williams/) |

---

### Dev Checklist
